src/components/pokedex.css

```diff
@@ -181,41 +182,42 @@
 .heart-button {
   position: absolute;
   top: -10px;
-  right: -200px; 
+  right: -190px;
   background: none;
   border: none;
   cursor: pointer;
+  border-radius: 50%;
+  padding: 10px;
   background-color: white;
 }
 
 .error-message {
   background-color: black;
   color: yellow;
-  padding: 10px; 
-  margin-top: 10px; 
-  border: 1px solid black; 
-  border-radius: 5px; 
+  padding: 10px;
+  margin-top: 10px;
+  border: 1px solid black;
+  border-radius: 5px;
   font-size: 12px;
 }
 
 .favorite-title {
   font-size: 2rem;
   font-weight: bold;
   align-self: center;
-  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
 }
 
 .favorite-pokemon-container {
-  display: flex;
-  flex-wrap: wrap;
-  align-items: flex-start;
-  margin-bottom: auto;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  width: 100%;
+  height: 100%;
 }
```

Comment:
this took me a little bit i did want to point you that there is a better solution for the **"3 rows of favorite pokemons, and each row has 2 items max"**

so, my recommendation here: everything related to "how to display in the screen" (in this case, this is a table of elements, 3 rows, 2 columns)

DO avoid Js , and try to solve that in pure CSS. 

In this case , the correct approach might be a Grid Layout

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout

https://css-tricks.com/snippets/css/complete-guide-grid/